testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/nativeauth/EmailSignInSignUpFragment.kt

```diff
@@ -108,66 +114,90 @@ class EmailSignInSignUpFragment : Fragment() {
     }
     private fun signIn() {
         CoroutineScope(Dispatchers.Main).launch {
-            try {
-                val email = binding.emailText.text.toString()
-
-                val actionResult = authClient.signIn(
-                    username = email
-                )
-
-                when (actionResult) {
-                    is SignInResult.CodeRequired -> {
-                        navigateToSignIn(
-                            signInstate = actionResult.nextState,
-                            codeLength =  actionResult.codeLength,
-                            sentTo = actionResult.sentTo,
-                            channel = actionResult.channel
+            val email = binding.emailText.text.toString()
+
+            System.out.println("1")
+            val actionResult = authClient.signIn(
+                username = email
+            )
+            System.out.println("2")
```

Comment:
Looks like some left over debugging here, same on line 119

Reply:
Sure. I will remove this line. We may not need test app any longer if the E2E tests are done.

Reply:
I think it will still be useful for manually debugging flows, and for checking incidents for on call engineers as well